code/modules/events/heart_attack.dm

```diff
@@ -0,0 +1,29 @@
+/datum/round_event_control/heart_attack
+	name = "Random Heart Attack"
+	typepath = /datum/round_event/heart_attack
+	weight = 20
+	max_occurrences = 2
+	earliest_start = 12000
+	min_players = 40 // To avoid shafting lowpop
+
+/datum/round_event/heart_attack/start()
+	for(var/mob/living/carbon/H in shuffle(GLOB.alive_mob_list))
```

Comment:
just like last time I believe this will make every human player have a heart attack.

Reply:
I don't follow, once it gives the disease it breaks the loop.

Reply:
Oh, you're right. This is really weird syntax, but I suppose it works. My bad.
On an unrelated note, using `GLOB.player_list` is probably cleaner here.

Reply:
Yeah i'm gonna redo this loop

Reply:
you want the intersect of both lists so you only infect alive players